Tests/scripts/clone_mocks.sh

```diff
@@ -5,4 +5,6 @@ ssh-keyscan github.com >> ~/.ssh/known_hosts
 
 if [[ ! -d "content-test-data" ]]; then
     git clone git@github.com:demisto/content-test-data.git
-fi
\ No newline at end of file
+  else
+    cd content-test-data && git reset --hard && git pull -r
```

Comment:
Why `git reset --hard`? Shouldn't the repo "be clean" when this is executed?

Reply:
It should, but in case we want to start from failed it won't be